packages/website/docs/examples/alert-block.md

```diff
@@ -212,25 +214,32 @@ export const Alert = (props: {
           })}
         </Menu.Dropdown>
       </Menu>
-      {/*Rich text field for user to type in*/}
-      <InlineContent style={inlineContentStyles} />
-    </div>
-  );
+    {/*Rich text field for user to type in*/}
+    <InlineContent style={{ flexGrow: "1" }} />
+  </div>
+);
 };
 
-// The Alert block itself
-export const AlertBlock = createReactBlockSpec({
-  type: "alert" as const,
-  propSchema: {
-    textAlignment: defaultProps.textAlignment,
-    textColor: defaultProps.textColor,
-    type: {
-      default: "warning",
-      values: ["warning", "error", "info", "success"],
-    },
-  } as const,
-  containsInlineContent: true,
-  render: Alert,
+// Function which creates the Alert block itself, where the component is styled
+// correctly with the light & dark theme
+  export const createAlertBlock = (theme: "light" | "dark") =>
```

Comment:
it would be cool if we have an example for fully themable custom blocks (now it just supports light / dark), but probably out of scope for now